fury/actor.py

```diff
@@ -2350,6 +2352,23 @@ def billboard(centers, colors=(0, 1, 0), scales=1, vs_dec=None, vs_impl=None,
     billboard_actor: Actor
 
     """
+    if using_gs:
+        bb_actor = dot(centers, colors)
+        replace_shader_in_actor(bb_actor, 'geometry',
+                                import_fury_shader('gs_billboard.geom'))
+        shader_to_actor(bb_actor, 'vertex',
+                        impl_code=import_fury_shader('gs_billboard_impl.vert'),
+                        decl_code=import_fury_shader('gs_billboard_dec.vert'),
+                        block='prim_id')
+
+        shader_to_actor(bb_actor, 'fragment',
+                        decl_code=import_fury_shader('gs_billboard_dec.frag'),
```

Comment:
Add support to the custom shaders in the params `fs_dec` and `fs_impl`. As implemented [here](https://github.com/fury-gl/fury/blob/2b2caf849efd2fa894d33be661a8dd3d6679c6b2/fury/actor.py#L2371) and [here](https://github.com/fury-gl/fury/blob/2b2caf849efd2fa894d33be661a8dd3d6679c6b2/fury/actor.py#L2373).